src/api/Migrations/20231211140724_AddCompletionTable.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace BDMS.Migrations;
+
+/// <inheritdoc />
+public partial class AddCompletionTable : Migration
+{
+    /// <inheritdoc />
+    protected override void Up(MigrationBuilder migrationBuilder)
+    {
+        migrationBuilder.CreateTable(
+            name: "completion",
+            schema: "bdms",
+            columns: table => new
+            {
+                id = table.Column<int>(type: "integer", nullable: false)
+                    .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn),
+                borehole_id = table.Column<int>(type: "integer", nullable: false),
+                is_primary = table.Column<bool>(type: "boolean", nullable: false),
+                name = table.Column<string>(type: "text", nullable: true),
+                kind_id_cli = table.Column<int>(type: "integer", nullable: false),
```

Comment:
Q/REQ: Weshalb braucht es dieses Feld noch? So wie ich es verstanden habe, wurde dieses Feld als Diskriminator verwendet, weil die Daten alle in die Stratigraphie-Tabelle gespeichert wurden. Siehe dazu auch https://github.com/geoadmin/suite-bdms/issues/254